translations/FOSUserBundle.da.yaml

```diff
@@ -0,0 +1,21 @@
+security:
```

Comment:
Consider using `bin/console translation:update` to generate translation files. Manually maintained (hierarchical) YAML files like this quickly become unmaintainable (as I see it). 

[Poedit](https://poedit.net) is a nice tool for editing [XLIFF files](https://en.wikipedia.org/wiki/XLIFF).

Reply:
This will be handled in a specific feature branch